app/konnect/org-management/sso.md

```diff
@@ -0,0 +1,175 @@
+---
+title: Configure Generic SSO
+---
+
+
+As an alternative to {{site.konnect_saas}}’s native authentication, you can set up single sign-on (SSO) access to {{site.konnect_short_name}} using OpenID Connect or SAML. This authentication method allows your users to log in to {{site.konnect_saas}} using their IdP credentials, without needing a separate login. This topic covers configuring SSO for use with various identity providers.
+
+We currently support the following providers: 
```

Comment:
Should this list of supported providers be placed under SAML? We really only have extended test coverage for these providers using SAML. Not that they can't work with OIDC, but we haven't tested them.